addon/adapters/pouch.js

```diff
@@ -351,7 +351,15 @@ export default DS.RESTAdapter.extend({
       queryParams.sort = this._buildSort(query.sort);
     }
 
-    return db.find(queryParams).then(pouchRes => db.rel.parseRelDocs(recordTypeName, pouchRes.docs));
+    var idType = type.documentType || recordTypeName;
+    var recordTypeNameRegex = new RegExp('^' + idType + '_[0-3]');
```

Comment:
I don't know if a regex is really the right way to go here. Feels a bit overkill. But using the normal way to parse the ID with parseDocID might also be too much. A _id.startsWith(idType + '_') might be enough here?

Reply:
oh, maybe babel does not have polyfills yet for startsWith by default? So the RegExp should probably be ok. Technically the idType should be escaped, but the characters that would need escaping would probably not be correct Ember Model names